src/engine/tests/Test_DisjunctionConstraint.h

```diff
@@ -205,23 +208,24 @@ class DisjunctionConstraintTestSuite : public CxxTest::TestSuite
             dc.notifyLowerBound( 2, -10 );
             dc.notifyUpperBound( 2, 10 );
 
-            TS_ASSERT( !dc.phaseFixed() );
+            TS_ASSERT( !dc.isImplication() );
```

Comment:
Do we want to also check phaseFixed() iff isImplication()?

Reply:
No the semantics actually differ. Disjunction does not ever explicitly fix the phase, it actually used a test that is exactly the implementation of isImplication. Therefore I removed the special implementation of DisjunctionConstraint::phaseFixed() and replaced the calls with isImplication().